### ISSUE TITLE
filterOnEventType - unexpected behaviour handling array of strings

### ISSUE THREAD

**Issue by vsnig** [jgilbert01/aws-lambda-stream#348]
I thought when providing an array of strings it should filter on exact match
But:
```javascript
const rule = { eventType: ['specific-thing-changed'] }
const uow = { event: { type: 'thing-changed' } }
expect(filterOnEventType(rule, uow)).toBe(false) <--- will fail
```
This is due to this check https://github.com/jgilbert01/aws-lambda-stream/blob/c678b09d63d6a5c8537fc6f5fe5660e743314ccb/src/filters/event.js#L10

I think it should be: 
```javascript
...
return rule.eventType.includes(uow.event.type);
...
```

**Comment by vsnig** [comment]
Just noticed that there's another issue addressing same problem #135

**Comment by vsnig** [comment]
Looks like its fixed by #385
